runtime/runtime/src/receipt_manager.rs

```diff
@@ -130,7 +129,7 @@ impl ReceiptManager {
         };
         let new_receipt_index = self.action_receipts.len() as ReceiptIndex;
         self.action_receipts.push((receiver_id.clone(), new_receipt));
-        self.yielded_data_ids.insert((receiver_id, input_data_id));
+        self.yielded_data_ids.insert((input_data_id, receiver_id));
```

Comment:
NB: this change just happened naturally as I tried out using hashmap for a previous revision. Too lazy to undo...